src/librustdoc/html/static/main.js

```diff
@@ -2362,6 +2361,7 @@ function defocusSearchBar() {
     (function() {
         var toggle = createSimpleToggle(false);
         var hideMethodDocs = getCurrentValue("rustdoc-auto-hide-method-docs") === "true";
+        var hideImplementors = getCurrentValue("rustdoc-auto-collapse-implementors") !== "false";
```

Comment:
Just curious - why do these use strings instead of `true` and `false`?

Reply:
When you use the local storage, it converts everything to string. As simple as that. :)